packages/test/test-version-utils/src/test/minCompatVersion.spec.ts

```diff
@@ -0,0 +1,87 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "assert";
+import { execSync } from "child_process";
+import { CompatKind } from "../../compatOptions.cjs";
+import { isCompatVersionBelowMinVersion } from "../compatConfig.js";
+
+describe("Minimum Compat Version", () => {
+	const allVersionsFromNpm = execSync(`npm show fluid-framework versions --json`, {
+		encoding: "utf-8",
+	});
+	const allVersions: string[] = JSON.parse(allVersionsFromNpm);
+	const latestVersion = allVersions[allVersions.length - 1];
```

Comment:
Is this really needed? Can't you just use `pkgVersion`?

Reply:
I tried using pkgVersion but it broke these tests when running in the pipeline. pkgVersion in ado is different than the one in repo for some reason. Trying with codeVersion but not 100% confident with that approach either